### PR TITLE
[feat] Check local token on initial load

### PR DESCRIPTION
**Pull request by RRDuke**

### CODE REVIEW

**Comment by codydaig** [comment]
Check Injection format before merging.

**Comment by codydaig** [comment]
LGTM

**Comment by trevorhanus** [comment]
:+1:

File: modules/core/client/app/app.js [hunk 1]
```diff
@@ -20,10 +20,44 @@ tropicalbs.config(function ($stateProvider, $urlRouterProvider) {
       abstract: true,
       controller: 'AppController'
     });
-});
 
-tropicalbs.config(function ($mdThemingProvider) {
+  $httpProvider.interceptors.push('AttachTokens');
+
+}])
+.factory('AttachTokens', ['$window', '$cookies', function ($window, $cookies) {
```
Comment (marcoviappiani) [inline modules/core/client/app/app.js:27]: fine for now but a lot of the things (config, factory for authentication) in this file should be split into a separate files

**Pull request merged**